frame/nomination-pools/src/lib.rs

```diff
@@ -1993,6 +2211,98 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Set the commission of a pool.
+		///
+		/// The dispatch origin of this call must be signed by the `root` role of
+		/// the pool. If the pool has a max commission set, the commission supplied
+		/// must be less or equal to that value.
+		///
+		/// If the max commission has not yet been set, then the commission range is
+		/// not bounded. A `payee` must be provided if commission has not yet been
+		/// set (still `None`). Once commission has been set, the `payee` can be
+		/// omitted in further calls. If a `payee` update is desired, the commission
+		/// must still be passed into the call.
+		#[pallet::weight(T::WeightInfo::set_commission())]
+		pub fn set_commission(
```

Comment:
Both `commission` and `payee` are now optional, and provided commission has already been set, a call can be made with only one or the other and succeed.